packages/apps/src/contexts/activity.ts

```diff
@@ -279,6 +292,26 @@ export class ActivityContext<T extends Activity = Activity> implements IActivity
     });
   }
 
+  private buildBlockQuoteForActivity(): string | null {
+    if (this.activity.type === 'message' && this.activity.text) {
+      const maxLength = 120;
+      const truncatedText =
+        this.activity.text.length > maxLength
+          ? `${this.activity.text.substring(0, maxLength)}...`
+          : this.activity.text;
+
+      return `<blockquote itemscope="" itemtype="http://schema.skype.com/Reply" itemid="${this.activity.id}">
+<strong itemprop="mri" itemid="${this.activity.from.id}">${this.activity.from.name}</strong><span itemprop="time" itemid="${this.activity.id}"></span>
+<p itemprop="preview">${truncatedText}</p>
+</blockquote>`;
+    } else {
+      this.log.debug('Skipping building blockquote for activity type:', this.activity.type);
+    }
+
+    return null;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
why are you ignoring this rule instead of putting the private method below?

Reply:
Hmm, I thought we wanted these to be at the bottom, stylistically, but I guess that was just a coincidence.